functions.php

```diff
@@ -1 +1,16 @@
 <?php
+
+function displayGameCard(array $allGames): string {
```

Comment:
the name implies that the function only displays 1 card for 1 game, whereas the function displays multiple cards for all the games, so the function needs a more semantic name